tutor/src/components/task-progress.js

```diff
@@ -132,12 +133,19 @@ class TaskProgress extends React.Component {
               if (!step.isInfo) {
                 progressIndex += 1;
                 return (
-                  <Cell
-                    key={stepIndex}
-                    className={cn({ 'current-step': step === currentStep })}
-                    onClick={() => goToStep(step.id)}>
-                    {progressIndex}
-                  </Cell>
+                  <>
+                    <Cell
+                      key={stepIndex}
+                      className={cn({ 'current-step': step === currentStep })}
+                      onClick={() => goToStep(step.id)}>
+                      {progressIndex}
+                      {step.isDroppedQuestion &&
+                      <CornerTriangle color="blue"
+                        tooltip={step.dropped_method == 'zeroed' ?
+                          'Question dropped: question is worth zero points' : 'Question dropped: full credit assigned for this question'}
+                      />}
+                    </Cell>
+                  </>
```

Comment:
doesn't seem like you need the fragment, `<>` since it has a single child Cell

Reply:
yep. nice cath @nathanstitt